apps/web/src/server/applications/case/representations/representation/check-answers/check-answers.controller.js

```diff
@@ -10,7 +10,7 @@ const view = 'applications/representations/representation/check-answers/index.nj
  * @param {import("express").Response} res
  */
 export const getCheckAnswersController = async (req, res) =>
-	res.render(view, getCheckAnswersViewModel(req.params.caseId, res.locals));
+	res.render(view, getCheckAnswersViewModel(req, res.locals));
```

Comment:
don't pass req as an argument. separate the web layer from the business logic.

Reply:
destrucutre the query / params at the controller level to follow all the other controllers.